### ISSUE TITLE
successful install then had to run 'evernode offerlease' after reboot

### ISSUE THREAD

**Issue by jellicoe** [EvernodeXRPL/evernode-test-resources#10]
Did a fresh install which was successful with the message: 

_Configuring Xahau message board service
Registering host on Evernode registry rnBfE4fDsypwptoBmVCQTgD4cKFLXr8ZTJ
Minted lease 3 of 3.
Installation successful! Installation log can be found at /tmp/evernode/installer-1707539099.log

Your system is now registered on Evernode. You can check your system status with 'evernode status' command._

then after reboot - ran 'evernode status' and got:

_Available Lease offers: 0 out of 3
Lease offers to be created: 3 out of 3
NOTE: Please use `evernode offerlease` command to create leases for the instances.

Host status: inactive_

ran evernode offerlease 

and leases / nfts created again 

node address https://explorer.xahau-test.net/r9wqFGZBXzwxdytdnGk21NMxbSXFahasPR/tx

---

looks like it did the URITokenMint in first run but not the URITokenCreateSellOffer

**Comment by chalith** [comment]
@jellicoe Yes we are now only doing the minting at installation so the host can choose to offer them or not. But we should add a message at the end mentioning that. Thanks for the suggesstion
